jgiven-junit5/src/main/java/com/tngtech/jgiven/junit5/JGivenExtension.java

```diff
@@ -35,61 +35,62 @@
  * @see SimpleScenarioTest
  */
 public class JGivenExtension implements
-        TestInstancePostProcessor,
-        BeforeAllCallback,
-        AfterAllCallback,
-        BeforeEachCallback,
-        AfterEachCallback {
+    TestInstancePostProcessor,
+    BeforeAllCallback,
+    AfterAllCallback,
+    BeforeEachCallback,
+    AfterEachCallback {
 
-    private static final Namespace NAMESPACE = Namespace.create( "com.tngtech.jgiven" );
+    private static final Namespace NAMESPACE = Namespace.create("com.tngtech.jgiven");
 
     private static final String REPORT_MODEL = "report-model";
 
     @Override
-    public void beforeAll( ExtensionContext context ) throws Exception {
+    public void beforeAll(ExtensionContext context) {
```

Comment:
So there was never an exception...?

Reply:
No that came in, because the interface Method declares an Exception. Which it needs to, because there could be an implementation somewhere, where a checked Exception is actually produced.